renv.lock

```diff
@@ -1,971 +1,958 @@
+      "Hash": "168827f4f5d1dbbaced99f4e6ca407b5"
     },
     "clipr": {
       "Package": "clipr",
-      "Version": "0.7.0",
-      "Source": "Repository",
-      "Repository": "CRAN",
-      "Hash": "9a99b54d2baf9393d0a68b6ea9513ee6"
-    },
-    "clisymbols": {
-      "Package": "clisymbols",
-      "Version": "1.2.0",
+      "Version": "0.7.1",
       "Source": "Repository",
       "Repository": "CRAN",
-      "Hash": "ce91c46caba29e30c44cfd87659bf8a4"
+      "Hash": "cf8a22309bd9bcb68c39c81054a020fa"
     },
     "colorspace": {
       "Package": "colorspace",
       "Version": "1.4-1",
       "Source": "Repository",
       "Repository": "CRAN",
-      "Hash": "c25d49739abb6ebf4b3d0c3565574226"
+      "Hash": "02ea402c2f83489444b41755f2ae0dd8"
     },
     "commonmark": {
       "Package": "commonmark",
       "Version": "1.7",
       "Source": "Repository",
       "Repository": "CRAN",
-      "Hash": "af7f698e0328e0474f794452177deca6"
+      "Hash": "0223fb58c32e5e77b963de822e9a5824"
     },
     "config": {
       "Package": "config",
       "Version": "0.3",
       "Source": "Repository",
       "Repository": "CRAN",
-      "Hash": "7a15075ad1798112b35de17b8de84dbc"
+      "Hash": "ec9c6b3de91f004a96c58fb08ff8569a"
     },
     "covr": {
       "Package": "covr",
-      "Version": "3.3.2",
+      "Version": "3.5.1",
+      "Source": "Repository",
+      "Repository": "CRAN",
+      "Hash": "6446639ffd5e708bd3d05bf1fcbeb3a7"
+    },
+    "cpp11": {
+      "Package": "cpp11",
+      "Version": "0.2.3",
       "Source": "Repository",
       "Repository": "CRAN",
-      "Hash": "1d82e0a7d7c413c607619004f96b9e3f"
+      "Hash": "e9908bd94c77dbb46ee829850649801d"
     },
     "crayon": {
       "Package": "crayon",
       "Version": "1.3.4",
       "Source": "Repository",
       "Repository": "CRAN",
-      "Hash": "d308ead32010b8e4d76c4690427da50b"
+      "Hash": "4fe720ae1ae43fcf231f0525edb0b134"
     },
     "crosstalk": {
       "Package": "crosstalk",
-      "Version": "1.0.0",
+      "Version": "1.1.0.1",
       "Source": "Repository",
       "Repository": "CRAN",
-      "Hash": "2bc962d85f29df259cc1a6930727ce87"
+      "Hash": "110a0ba9f8ae339221f1b054fa45ce24"
     },
     "curl": {
       "Package": "curl",
       "Version": "4.3",
       "Source": "Repository",
       "Repository": "CRAN",
-      "Hash": "f42ae4b12d342a357cc551ca1ecd8cc1"
+      "Hash": "5168f99143001162086cb407847bc913"
     },
     "dccvalidator": {
       "Package": "dccvalidator",
-      "Version": "0.0.0.9011",
-      "Source": "GitHub",
-      "RemoteType": "github",
-      "RemoteHost": "api.github.com",
-      "RemoteUsername": "Sage-Bionetworks",
-      "RemoteRepo": "dccvalidator",
-      "RemoteRef": "master",
-      "RemoteSha": "531375ccbc2e323d2c2f5bc4d70b02880c6b362d",
-      "Hash": "0efbac7c78190be952b066c0a4bcc47b"
+      "Version": "0.3.0.9010",
+      "Source": "Repository",
+      "Repository": "CRAN",
```

Comment:
Something is wrong here; dccvalidator 0.3.0.9010 is not on CRAN, so this fails to install when I do `renv::restore()`

Reply:
Sorry! Fixed.